R/analysis_gradient_flow.R

```diff
@@ -7,14 +7,14 @@ gf.scales[["sqrt_t0_Esym"]] <- list(val=rep(0,3), ref.val=0.1416, ref.dval=0.000
 gf.scales[["w0_Wsym"]] <- list(val=rep(0,3), ref.val=0.1755, ref.dval=0.0019, approx.idx=0,
                                label="\\w_0^{\\mathrm{sym}}", obs="Wsym")
 
-analysis_gradient_flow <- function(path,basename,read.data=TRUE,plot=FALSE,skip=0,start=0,scale=1,dbg=FALSE) {
+analysis_gradient_flow <- function(path,outputbasename,basename="gradflow",read.data=TRUE,plot=FALSE,skip=0,start=0,scale=1,dbg=FALSE) {
   if(missing(basename)){
```

Comment:
and 'basename' will never be missing...?

Reply:
indeed, this can be removed I guess.